docs/sources/enterprise/license-restrictions.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "License restrictions"
+description = "Grafana Enterprise license restrictions"
+keywords = ["grafana", "licensing", "enterprise"]
+weight = 8
++++
+
+# License restrictions
+
+Enterprise licenses are limited by the number of active users, a license expiration date, and the URL of the Grafana instance.
+
+**User limits**
+
+Grafana licenses allow a certain number of active users in an instance. An active user is any user that has signed in to Grafana within the past 30 days.
+
+In the context of licensing, each user is classified as either a viewer or an editor, which are defined as follows:
+
+- An editor is a user who has the ability to edit and save dashboards, which includes:
+    - Grafana server admins
+    - Users assigned an Org role of "Editor" or "Admin," 
+    - Users that have been granted "Admin" or "Edit" permissions through [dashboard/folder permissions](https://grafana.com/docs/grafana/latest/permissions/dashboard_folder_permissions/).     
+- A viewer is a user with the Viewer role who does not have the permissions to save dashboards.
+
+Restrictions are applied separately for viewers and editors.
+
+When the number of maximum active viewers or editors is reached, a warning banner is displayed within Grafana.
+
+**Expiration date**
+
+The license expiration date is the day when a license is no longer active. A banner will appear in Grafana Enterprise when the license expiration date approaches.
```

Comment:
```suggestion
The license expiration date is the date when a license is no longer active. As the license expiration date approaches, Grafana Enterprise displays a banner.
```

Technically, a date is 01, 02, 14, etc. and they day is Monday, Tuesday, Wednesday, etc.

Reply:
@oddlittlebird What is our convention for "is displayed" versus "appears"? Never mind, found it here: https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#display-verb